password_renewal_bp.py

```diff
@@ -41,29 +46,29 @@ def show_heslonew(token):
 			#print(token_data)
 		except InvalidToken:
 			print("Neplatný token.")
-			flash ("Neplatný link.", "danger")
+			flash("Neplatný link.", "danger")
 		return render_template('noveheslo.html', token=token)
 
 
 @blueprint.route('/noveheslo', methods=['POST'])
 def heslo_new():
 	result = request.form
-	chyba = None
+	error = None
 	token = result.get("token")
-	id_uzivatele = json.loads(f.decrypt(token.encode("ascii"), ttl=60*60*24))['id']
-	heslo = result.get("heslo")
-	heslo_potvrzeni = result.get("heslo_potvrzeni")
-	if not heslo == heslo_potvrzeni:
-		flash ('Hesla se neshodují.', "danger")
+	id_user = json.loads(f.decrypt(token.encode("ascii"), ttl=60*60*24))['id']
+	password = result.get("password")
```

Comment:
the string in quotes should remain "heslo", unless you remain the form field in the registrace.html template as well. It's ok to change the variable like this.